startup.sh

```diff
@@ -1,49 +1,17 @@
 #!/bin/bash
 
-# load kernel module
-cd ~pi/xmaspi/lkm
-insmod xmas.ko
-mknod /dev/xmas c 252 0
-chmod 222 /dev/xmas
-
-# run the init script
-cd ~pi/xmaspi
-./driver.py
-
-
-# red status
-./driver.py 99 200 0 0 13
+echo Loading kernel module
 
-# start wireless
-./start-wireless.sh&
+( cd lkm ; ./setup.sh )
 
+echo Initializing strand
 
-# wait for wifi...
-ifconfig | grep 192.168
-r=`echo $?`
-while [[ $r -eq 1 ]];
-do
-    ./all_on.py 0 0 0 0
-    sleep .5
-    ./all_on.py 128 13 0 0
-    sleep .5
-
-    ifconfig | grep 192.168
-    r=`echo $?`
-    
-done
+./driver.py
 
-for i in `seq 1 5`
-do
-    ./all_on.py 0 0 0 0
-    sleep .1
-    ./all_on.py 100 0 13 0
-    sleep .1
-done
+echo Launching UDP driver
 
-ntpdate -ub 0.us.pool.ntp.org
```

Comment:
Does the RPI do this automatically now? I remember having issues with it not having accurate time and otherwise valid certs being marked as invalid.

Reply:
Yes, ntp works correctly out of the box now.